ui/src/app/pages/admin/user/admin-user-profile.tsx

```diff
@@ -231,26 +232,37 @@ const InitialCreditsCard = ({
       <FlexRow style={styles.initialCreditsPanel}>
         <FlexColumn>
           <div style={styles.subHeader}>Initial credits</div>
-          <InstitutionExpirationBypassExplanation
-            bypassed={
-              !!institution?.institutionalInitialCreditsExpirationBypassed
-            }
-          />
-
-          <FlexColumn>
-            {enableInitialCreditsExpiration && (
-              <InitialCreditBypassSwitch
-                currentlyBypassed={
-                  updatedProfile.initialCreditsExpirationBypassed
-                }
-                previouslyBypassed={oldProfile.initialCreditsExpirationBypassed}
-                expirationEpochMillis={
-                  oldProfile.initialCreditsExpirationEpochMillis
-                }
-                onChange={(bypass) => onChangeInitialCreditBypass(bypass)}
-                label='Individual Expiration Bypass'
-              />
+          {enableInitialCreditsExpiration && (
```

Comment:
Unrelated change, but the institution bypass logic should only be shown if initial credit expiration is enabled. I can move this to a separate PR, if you would prefer.